src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -21,19 +22,31 @@ const MAX_VERSIONS = DEFAULT_FETCH_END;
 
 type Props = {
     error?: MessageDescriptor,
+    errorTitle?: MessageDescriptor,
     fileId: string,
     isLoading: boolean,
+    onUpgradeCTAClick: () => void,
```

Comment:
This is an optional prop, right?

```suggestion
    onUpgradeCTAClick?: () => void,
```